plot_goes_x.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+import argparse
+import matplotlib
+matplotlib.use('agg')
+import matplotlib.pyplot as plt
+import tables
+from Chandra.Time import DateTime
+
+from Ska.Matplotlib import plot_cxctime
+
+parser = argparse.ArgumentParser(description='')
+parser.add_argument('--out', type=str,
+                    default='goes_x.png',
+                    help='Plot file name')
+parser.add_argument('--h5',
+                    default='GOES_X_16.h5',
+                    help='HDF5 file name')
+args = parser.parse_args()
+
```

Comment:
Only 1 vertical whitespace line here.  Your default code editor should be complaining at this.

Reply:
I'm in editor hell.